TAF/testScenarios/integrationTest/UC_north_south_messaging/GET-Negative.robot

```diff
@@ -0,0 +1,66 @@
+*** Settings ***
+Resource        TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource        TAF/testCaseModules/keywords/device-sdk/deviceServiceAPI.robot
+Suite Setup     Setup Suite
+Suite Teardown  Run Teardown Keywords
+Force Tags      Skipped
+
+*** Variables ***
+${SUITE}          North-South Messaging GET Negative Testcases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/north-south-messaging-get-negative.log
+
+*** Test Cases ***
+ErrNSMessagingGET001 - Query all DeviceCoreCommands with invalid offset
+    Given Subscribe MQTT Topic 'edgex/commandquery/response'
+    When Query Commands With invalid offset For All Devices From External MQTT Broker
+    Then Should Return Error Code "1"
+    And RequestID Should Be Correct
+
+ErrNSMessagingGET002 - Query Commands with non-existent device name
+    Given Subscribe MQTT Topic 'edgex/command/response/#'
+    When Get Command With non-existent device From External MQTT Broker
```

Comment:
I'm confused whether this is a query command or a device command ...
so I think before the AdminState lock case we need:
- Query all with invalid limit/offset (case1)
- Query by device name with non-exist device name (case2 ...?)
- Get device command with non-exist device name (case3)
- Get device command with non-exist command name

Reply:
It should be query command. Updated the error and added more cases for the function.